examples/sveltekit-live-avatars/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "sveltekit-pixel-art",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "svelte-kit dev",
+    "build": "svelte-kit build",
+    "package": "svelte-kit package",
+    "preview": "svelte-kit preview",
+    "check": "svelte-check --tsconfig ./tsconfig.json",
+    "check:watch": "svelte-check --tsconfig ./tsconfig.json --watch"
+  },
+  "devDependencies": {
+    "@sveltejs/adapter-auto": "next",
+    "@sveltejs/kit": "next",
+    "@types/cookie": "^0.4.1",
+    "autoprefixer": "^10.4.2",
+    "postcss": "^8.4.6",
+    "svelte": "^3.46.0",
+    "svelte-check": "^2.2.6",
+    "svelte-preprocess": "^4.10.1",
+    "tailwindcss": "^3.0.23",
```

Comment:
Are we using tailwindcss only for the example info box? I'm wondering if we should build it with in css only to reduce the noise for developers that will clone the example.